specs/sharding/beacon-chain.md

```diff
@@ -103,460 +101,342 @@ The following values are (non-configurable) constants used throughout the specif
-| `POINTS_PER_SAMPLE` | `uint64(2**3)` (= 8) | 31 * 8 = 248 bytes |
+| `POINTS_PER_SAMPLE` | `uint64(2**4)` (= 16) | 31 * 16 = 496 bytes |
 | `MODULUS` | `0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001` (curve order of BLS12_381) |
 
-### Domain types
-
-| Name | Value |
-| - | - |
-| `DOMAIN_SHARD_BLOB` | `DomainType('0x80000000')` |
+## Preset
 
-### Shard Work Status
+### Misc
 
 | Name | Value | Notes |
 | - | - | - |
-| `SHARD_WORK_UNCONFIRMED` | `0` | Unconfirmed, nullified after confirmation time elapses |
-| `SHARD_WORK_CONFIRMED` | `1` | Confirmed, reduced to just the commitment |
-| `SHARD_WORK_PENDING` | `2` | Pending, a list of competing headers |
-
-### Misc
-
-TODO: `PARTICIPATION_FLAG_WEIGHTS` backwards-compatibility is difficult, depends on usage.
-
-| Name | Value |
-| - | - |
-| `PARTICIPATION_FLAG_WEIGHTS` | `[TIMELY_SOURCE_WEIGHT, TIMELY_TARGET_WEIGHT, TIMELY_HEAD_WEIGHT, TIMELY_SHARD_WEIGHT]` |
-
-### Participation flag indices
+| `MAX_SHARDS` | `uint64(2**12)` (= 4,096) | Theoretical max shard count (used to determine data structure sizes) |
+| `ACTIVE_SHARDS` | `uint64(2**8)` (= 256) | Initial shard count |
+| `TARGET_SHARDS` | `uint64(ACTIVE_SHARDS // 2)` (= 256) | Initial shard count |
 
-| Name | Value |
-| - | - |
-| `TIMELY_SHARD_FLAG_INDEX` | `3` |
+### Time parameters
 
-### Incentivization weights
-
-TODO: determine weight for shard attestations
-
-| Name | Value |
-| - | - |
-| `TIMELY_SHARD_WEIGHT` | `uint64(8)` |
+With the introduction of intermediate blocks the number of slots per epoch is doubled (it counts beacon blocks and intermediate blocks).
 
-TODO: `WEIGHT_DENOMINATOR` needs to be adjusted, but this breaks a lot of Altair code.
-
-## Preset
-
-### Misc
-
-| Name | Value | Notes |
-| - | - | - |
-| `MAX_SHARDS` | `uint64(2**10)` (= 1,024) | Theoretical max shard count (used to determine data structure sizes) |
-| `INITIAL_ACTIVE_SHARDS` | `uint64(2**6)` (= 64) | Initial shard count |
-| `SAMPLE_PRICE_ADJUSTMENT_COEFFICIENT` | `uint64(2**3)` (= 8) | Sample price may decrease/increase by at most exp(1 / this value) *per epoch* |
-| `MAX_SHARD_PROPOSER_SLASHINGS` | `2**4` (= 16) | Maximum amount of shard proposer slashing operations per block |
-| `MAX_SHARD_HEADERS_PER_SHARD` | `4` | |
-| `SHARD_STATE_MEMORY_SLOTS` | `uint64(2**8)` (= 256) | Number of slots for which shard commitments and confirmation status is directly available in the state |
-| `BLOB_BUILDER_REGISTRY_LIMIT` | `uint64(2**40)` (= 1,099,511,627,776) | shard blob builders |
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `SLOTS_PER_EPOCH` | `uint64(2**6)` (= 64) | slots | 8:32 minutes |
 
 ### Shard blob samples
 
 | Name | Value | Notes |
 | - | - | - |
-| `MAX_SAMPLES_PER_BLOB` | `uint64(2**11)` (= 2,048) | 248 * 2,048 = 507,904 bytes |
-| `TARGET_SAMPLES_PER_BLOB` | `uint64(2**10)` (= 1,024) | 248 * 1,024 = 253,952 bytes |
+| `SAMPLES_PER_BLOB` | `uint64(2**9)` (= 2,048) | 248 * 2,048 = 507,904 bytes |
```

Comment:
@dankrad I thought we're doing 128 kB per blob?

Reply:
Correct, the comment was wrong.